protected/gii/CaseSearchParameter/templates/default/case_search_parameter.php

```diff
@@ -58,6 +59,8 @@ public function renderParameter($id)
 
     }
 
+<?php foreach(explode(',', $this->searchProviders) as $searchProvider):?>
+<?php if ($searchProvider === 'DBProvider'):?>
```

Comment:
I might be able to work around this by adding the function declarations as an input field. Not sure how well that will work with the case search parameter template though.